examples/aggregates/README.md

```diff
@@ -1,3 +1,9 @@
+<!---
+  Copyright 2023 SECO Mind Srl
+
+  SPDX-License-Identifier: LGPL-2.1-or-later or Apache-2.0
```

Comment:
I'm not sure if SPDX accepts lowercase `or`, so maybe we can just use `LGPL-2.1-or-later OR Apache-2.0`.